static/api/extensions/redhat/bootc/1.14.0/README.md

```
@@ -0,0 +1,351 @@
+# BootC (Bootable Container) Extension for Podman Desktop
+
+![](https://raw.githubusercontent.com/podman-desktop/extension-bootc/main/docs/img/banner.png)
```

markdownlint-cli2 (0.22.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)